book/src/ephemeral-local-volumes.md

```diff
@@ -2,15 +2,44 @@
 
 ## Status
```

Comment:
should we remove "local" from the page name?

Reply:
That would also change the URL of the page. While I agree that the page name isn't ideal, I'm leaning towards keeping it to avoid breaking links. It only shows up in URLs, so it's not that visible.